vmhost/hostCore/host.go

```diff
@@ -580,31 +572,6 @@ func (host *vmHost) EpochConfirmed(epoch uint32, _ uint64) {
 	}
 }
 
-// FixOOGReturnCodeEnabled returns true if the corresponding flag is set
-func (host *vmHost) FixOOGReturnCodeEnabled() bool {
```

Comment:
q: why are all these flags removed?

Reply:
Not needed. Not used, should have been cleaned before.

Reply:
ok, so unrelated with this PR, just some extra cleanup, got it 👍